Why shouldn't Macs be allowed to use the "system" (MacPorts) copy of PCRE?
--- 3rdparty/CMakeLists.txt.orig	2017-01-04 10:45:26.000000000 -0600
+++ 3rdparty/CMakeLists.txt	2017-11-19 12:29:42.000000000 -0600
@@ -31,7 +31,7 @@
     MESSAGE(STATUS "Using System Hunspell")
 endif()
 
-if( NOT USE_SYSTEM_LIBS OR NOT PCRE2_FOUND OR APPLE )
+if( NOT USE_SYSTEM_LIBS OR NOT PCRE2_FOUND )
     MESSAGE(STATUS "Using Bundled PCRE2")
     include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/pcre2.cmake)
 else ()
@@ -39,8 +39,7 @@
 endif()
 
 if (SYSTEM_LIBS_REQUIRED)
-    if (NOT ZLIB_FOUND OR NOT MINIZIP_FOUND OR NOT HUNSPELL_FOUND OR (NOT APPLE AND NOT PCRE2_FOUND))
-        #if (NOT ZLIB_FOUND OR NOT MINIZIP_FOUND OR (NOT APPLE AND NOT PCRE2_FOUND))
+    if (NOT ZLIB_FOUND OR NOT MINIZIP_FOUND OR NOT HUNSPELL_FOUND OR NOT PCRE2_FOUND)
         MESSAGE(FATAL_ERROR "Could not find all required system libraries...")
     endif()
 endif()
--- CMakeLists.txt.orig	2017-03-21 14:01:47.000000000 -0500
+++ CMakeLists.txt	2017-11-19 12:36:28.000000000 -0600
@@ -99,9 +99,7 @@
     find_package( PkgConfig )
     find_package( MiniZip )
     pkg_check_modules( HUNSPELL hunspell )
-    if ( NOT APPLE )
-        pkg_check_modules( PCRE2 libpcre2-16 )
-    endif()
+    pkg_check_modules( PCRE2 libpcre2-16 )
 endif()
 
 message(STATUS "CMake version in use: ${CMAKE_VERSION}")
